packages/web/src/pages/modals/Modals.tsx

```diff
@@ -91,6 +90,7 @@ const Modals = () => {
       <UnloadDialog />
       <CollectibleDetailsModal />
       <StripeOnRampModal />
+      <CreateChatModal />
```

Comment:
Don't unmount the modal when hiding it so it retains state

Reply:
Will this save state if user dismisses and then later comes back? ie. are you clearing state on dismiss?

Reply:
I see clearing the query, close vs dismiss getting confusing